charts/kube-prometheus-stack/charts/crds/templates/upgrade/clusterrole.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.upgradeJob.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "kube-prometheus-stack.upgradeCRD.name" . }}
+  namespace: {{ template "kube-prometheus-stack.namespace" . }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade,pre-rollback
```

Comment:
Does that mean that this will rollback the crds successfully? That would be pretty neat but if not, it probably needs to be documented in the upgrade process

Reply:
theoretically yes.